content/news/ragls5-and-team-tournament.md

```diff
@@ -0,0 +1,20 @@
+<div style="text-align:center" markdown="1">
+![Red Alert Global League Season 5](/images/news/20180317-ragl-V.png)
+</div>
+
+Greetings, commanders!
+
+Red Alert Global League is rolling yet again, with Season 5 starting on April 2nd. Masters division is as strong as ever this time, with several players returning to top tier and new promising contestants rising up as well; Minions are split into 2 groups as the single division wasn't able to hold that many good players. To raise the stakes even higher the map pool now holds mostly new competitive maps so that players are challenged to improvise and develop new tactics and strategies on the battlefields.
+
+New players are always welcome to register and take part in the League. It is always exciting to prove your worth against others in a fair competition, and there is also a chance to get some prizes through playing. Be swift on the matter though, as the registrations are to be closed on March 24th.
+
+League rules can be found at [ragl.org forum](http://ragl.org/glforum/?view=thread&id=12&part=1#postid-22) and [as a googlesheet mirror](https://goo.gl/VDwMNm). 
+Registration guidelines are at [ragl.org forum](http://ragl.org/glforum/?view=thread&id=11&part=1#postid-19) and [on sleipnirstuff](http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=20470). 
+You can also ask anything about the League, find opponents for a match and have a casual chat at [community discord server](https://discord.gg/UcUztfr).
+
+
+If you don't like small tactical action tet-a-tet, but rather prefer to feel the support of your teammate as well as the heat of battle, there is a new 2v2 Team Tournament starting soon just for you! Two-stage tournament played from April 4th till April 22nd is already gathering some big names around and will definitely be a nice show to observe and participate.
```

Comment:
Looks like this is actually spelled `tête-à-tête` or `tete-a-tete` in english (I only found it that way in a [quick search](https://en.wiktionary.org/wiki/t%C3%AAte-%C3%A0-t%C3%AAte)).

Also, what about `The two-stage tournament played [...]`? Just "Two-stage tournament played from x till y is gathering" sounds weird to me.